_re2.cc

```diff
@@ -216,14 +249,6 @@ static PyTypeObject Regexp_Type2 = {
   regexp_methods,              /*tp_methods*/
   0,                           /*tp_members*/
   0,                           /*tp_getset*/
-  0,                           /*tp_base*/
```

Comment:
Why remove these?

Reply:
By default all omitted entries are 0 and since we're no longer setting the attr_dict I thought i'd trim down the list to be consistent with the other typedef in the source. I can zero the fields out and leave it as is though if that's preferred.

Reply:
Yeah, I'd leave it.  Explicit is better than implicit.